ydb/topic.py

```diff
@@ -43,11 +47,25 @@
 
 
 class TopicClientAsyncIO:
+    _closed: bool
     _driver: aio.Driver
     _credentials: Union[Credentials, None]
+    _settings: TopicClientSettings
+    _executor: concurrent.futures.Executor
 
-    def __init__(self, driver: aio.Driver, settings: "TopicClientSettings" = None):
+    def __init__(self, driver: aio.Driver, settings: Optional[TopicClientSettings]):
+        if not settings:
+            settings = TopicClientSettings()
+        self._closed = False
         self._driver = driver
+        self._settings = settings
+        self._executor = concurrent.futures.ThreadPoolExecutor(
```

Comment:
It will need in reader for decode messages in https://github.com/ydb-platform/ydb-python-sdk/issues/135

By default the executor must share between all readers/writers, created from one Client or Driver.